src/databricks/labs/ucx/source_code/known.json

```diff
@@ -32815,6 +32815,21 @@
     "trl.trainer.sft_trainer": [],
     "trl.trainer.utils": []
   },
+  "typeguard": {
```

Comment:
See this CI: https://github.com/databrickslabs/ucx/actions/runs/12651632798/job/35252615103